controllers/policy.go

```
@@ -565,8 +538,8 @@ func (r *ClusterGroupUpgradeReconciler) newBatchPlacementBinding(clusterGroupUpg
 		},
 		"placementRef": map[string]interface{}{
 			"name":     placementName,
-			"kind":     utils.PlacementGVK.Kind,
-			"apiGroup": utils.PlacementGVK.Group,
+			"kind":     "Placement",
+			"apiGroup": clusterv1beta1.SchemeGroupVersion.Group,
```

golangci-lint (2.12.2)

[error] SA1019: clusterv1beta1.SchemeGroupVersion is deprecated: use GroupVersion instead. 

(staticcheck)